Disable most of the TinyMCE buttons

johngodley opened the pull request:
TinyMCE loads a lot of formatting tools that aren't available when editing on bbPress or other systems. TinyMCE will be used when editing old content.

This PR disables most of the toolbar buttons.

Fixes #44

![image](https://user-images.githubusercontent.com/1277682/204303126-a220c5be-e18b-498c-bd04-2bcbe3649ebc.png)

johngodley committed:
Disable most of the TinyMCE buttons

We just want the basics

dd32 commented:
This looks good to me, for fixing #44.

I do ponder if making the change as global as being done here is going to be correct for other use-cases, and if this shouldn't partially triggered by the bbPress integration.

Note: The screenshot here indicates that toolbar2 would be hidden, ie. `$settings['toolbar2'] = '';` where as the code includes the second line with the paste buttons/etc. Once again, this is probably integration specific as to whether it's worth hiding or not.

johngodley committed:
Move tinymce mod

Move it a bit later, to help prevent it running

Also, move all the buttons into the first toolbar

johngodley commented:
The changes should only run when the bbPress handler is running, but I've tweaked it a bit so it's specifically done when the plugin is doing it's editor stuff. I think this should cover most situations, and if not I can probably move the code to JS so it runs just before showing the editor.

Yeah, I see what you mean about the toolbar. I don't know how TinyMCE works, and whatever magic is needed for toolbar2 is escaping me, so I've put them all on the first line. This probably makes more sense anyway as there's plenty of space.
